tests/SignupApiTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class SignupApiTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    /*
+     * Base URL for the Api.
+     */
+    protected $signupsApiUrl = 'api/v2/signups';
+
+    /**
+     * Test that a POST request to /signupss creates a new signup.
+     *
+     * @group creatingAPhoto
+     * @return void
+     */
+    public function testCreatingASignup()
+    {
+    	// Create test signup
+        $signup = [
+            'northstar_id'     => str_random(24),
+            'campaign_id'      => $this->faker->randomNumber(4),
+            'campaign_run_id'  => $this->faker->randomNumber(4),
+            'source'           => 'the-fox-den',
+            'do_not_forward'   => true,
+        ];
+
```

Comment:
We have a model factory for a signup if you want use [here] (https://github.com/DoSomething/rogue/blob/master/database/factories/ModelFactory.php#L30) and add anything that is missing! Then you can just call `$signup = factory(Signup::class)->create();` here.